apps/chat/src/utils/app/import-export.ts

```diff
@@ -394,12 +394,14 @@ export const updateAttachment = ({
     return oldAttachment;
   }
 
-  const oldAttachmentId = decodeApiUrl(oldAttachmentUrl);
+  const oldAttachmentDecodedUrl = decodeApiUrl(oldAttachmentUrl);
```

Comment:
you can revert this line, because `constructPath(parentPath, name)` equals `oldAttachmentDecodedUrl`
```suggestion
  const oldAttachmentId = decodeApiUrl(oldAttachmentUrl);
```
and remove `const oldAttachmentId = constructPath(parentPath, name);`

Reply:
> constructPath(parentPath, name) equals oldAttachmentDecodedUrl

This is not quite right.
oldAttachmentDecodedUrl  includes bucket and Apikey('files') addresses
and oldAttachmentId not includes them. 
The problem here is rather in wrong naming.
I have renamed **oldAttachmentId** to **oldAttachmentRelativePath** to avoid misunderstandings in future.